moveit_core/trajectory_processing/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
 set(MOVEIT_LIB_NAME moveit_trajectory_processing)
 
+find_package(ruckig REQUIRED)
```

Comment:
This is a `moveit_core` dependency now, please move into the top-level CMakeLists.txt along with the other `find_package()` calls. (I see you already added ruckig to `THIS_PACKAGE_INCLUDE_DEPENDS`, also).